nft/nfts.yaml

```diff
@@ -754,6 +754,40 @@ paths:
                           type: integer
                           description: Integer - The index of the token within the bundle of NFTs sold in the sale.
       operationId: getNFTSales
+  '/{apiKey}/getContractsForOwner':
+    get:
+      summary: getContractsForOwner
+      description: Gets all NFT contracts held by an owner address.
+      tags: ['NFT API Endpoints']
+      parameters:
+        - $ref: '#/components/schemas/apiKey'
+        - $ref: '#/components/schemas/owner'
+        - $ref: '#/components/schemas/pageKey'
+      x-readme:
+        samples-languages:
+          - javascript
+          - curl
+          - python
+          - go
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  ownedNfts:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/ownedContract'
+                  pageKey:
+                    type: string
+                    description: 'String - UUID for pagination - returned if there are more NFTs contracts to fetch. Max NFT contracts per page = 100.'
```

Comment:
@ayang-rkjs we already have pageKey defined in components with a slightly different definition. Can we adjust these to match and use the same definition?